pybotx_widgets/checktable.py

```diff
@@ -27,7 +26,7 @@ class Config:
         arbitrary_types_allowed = True
 
     @root_validator(pre=True)
-    def is_checkbox_value_exist_in_mapping(cls, values):
+    def is_checkbox_value_exist_in_mapping(cls, values: dict):
```

Comment:
It's 3.9 feature, pls use `typing.Dict`.

Reply:
done